README.md

```diff
@@ -1,18 +1,33 @@
 # rack
-An opinionated CLI for Rackspace interactions
+The official command line interface for the Rackspace Cloud.
+
+`rack` provides a consistent interface for interacting with the Rackspace Cloud. For example, creating a new server:
+
+```
+$ rack servers instance create
```

Comment:
Can we add some `\` at the end of the lines of these commands for users who try to copy-and-paste it?